lucene/core/src/java/org/apache/lucene/codecs/lucene80/Lucene80DocValuesConsumer.java

```diff
@@ -962,6 +1078,21 @@ public SortedDocValues getSorted(FieldInfo field) throws IOException {
     addressesWriter.finish();
     meta.writeLong(data.getFilePointer() - start); // addressesLength
 
-    addTermsDict(values);
+    long valuesCount = values.getValueCount();
```

Comment:
Could we factor this code? It is nearly identical to the new code in doAddSortedField().

Reply:
will remove this:)